.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

I think it's worth having a daily check rather than once a week
```suggestion
      interval: "daily"
```